@xen-orchestra/web-core/lib/components/UserLogo.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <span :class="size" class="user-logo" />
+</template>
+
+<script lang="ts" setup>
+import type { UserLogoSize } from '@core/types/size.type'
+
+withDefaults(
+  defineProps<{
+    size?: UserLogoSize
+  }>(),
+  { size: 'extra-small' }
+)
+</script>
+
+<style lang="postcss" scoped>
+/* COLOR VARIANTS */
+.user-logo {
+  --border-color: var(--color-purple-base);
+}
+
+/* SIZE VARIANTS */
+.user-logo {
+  &.extra-small {
+    --size: 1.6rem;
+    --border-size: 0.1rem;
+    --background-position: top -0.493rem left -0.8rem / 2.88rem;
```

Comment:
I'm picky, but I'd prefer to separate the size from the position.

Also, `background-position` can be shorter.

```suggestion
    --background-position: -0.8rem -0.493rem;
    --background-size: 2.88rem;
```